content/docs/gto/user-guide/downstream.md

```diff
@@ -0,0 +1,40 @@
+# Acting downstream
+
+Once Git tag was pushed, you can start acting downstream. One of the popular
+options here is to use CI/CD, setting it to be triggered by pushing a Git tag.
+For general details, check out something like
+[GitHub Actions](https://github.com/features/actions),
+[GitLab CI/CD](https://docs.gitlab.com/ee/ci/) or
+[Circle CI](https://circleci.com).
+
+The other option is to
+[configure webhooks](https://docs.github.com/en/rest/webhooks) that will send
+HTTP requests to your server upon pushing Git tags to the remote.
+
+Finally, you can configure your server to query your Git provider via something
+like REST API to check if changes happened. As an example, check out
+[Github REST API](https://docs.github.com/en/rest).
+
+## Getting started with CI/CD
+
+To act upon registrations and assignments (Git tags), you can create simple CI
+workflow. Check out
+[the example workflow in `example-gto` repo](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml).
+The workflow uses [the GTO GH Action](https://github.com/iterative/gto-action)
+that fetches all Git tags (to correctly interpret the Registry) and finds out
+the version of the artifact that was registered, or the stage that was assigned,
+so you could use them in later steps of the CI.
+
+## Inspecting Git tags
+
+You can use `gto check-ref` to interpret the Git tag:
+
+```console
+$ gto check-ref -r build/example-gto churn#prod#3
+✅  Stage "prod" was assigned to version "v3.0.0" of artifact "churn"
+```
+
+For machine-consumable format, use `--json` flag or output specific pieces of
+information with `--name`, `--version`, `--stage` or `--event`.
+
+TODO: add `gto describe` here.
```

Comment:
@jorgeorpinel, I'm tempted to make this a small tutorial and add tell about `describe` as well. Pros: it's a small tutorial. Cons: with this approach, Command reference will duplicate some info.

I'm thinking if we need command-reference at all here TBH. From one side, `gto cmd -h` have even command examples and I kinda like them here - they can help when they're quickly reachable from CLI, e.g.

<details>

```shell
$ gto register -h
Usage: gto register [OPTIONS] NAME [REF]

  Create an artifact version to signify an important, published or released
  iteration

Arguments:
  NAME   Artifact name  [required]
  [REF]  Git reference to use for registration  [default: HEAD]

Options:
  -r, --repo TEXT        Local or remote repository  [default: .]
  --version, --ver TEXT  Version name in SemVer format
  -m, --message TEXT     Message to annotate git tag with
  --simple TEXT          [auto, true, false] Use simple notation, e.g. rf#prod
                         instead of rf#prod-5  [default: auto]
  --force                Create a git tag even if it already exists and is in
                         effect
  --bump-major           Bump major version
  --bump-minor           Bump minor version
  --bump-patch           Bump patch version
  --push                 Push created tag automatically (experimental)
  -h, --help             Show this message and exit.

Examples:
    Register new version at HEAD:
    $ gto register nn

    Register new version at a specific ref:
    $ gto register nn abc1234

    Assign version name explicitly:
    $ gto register nn --version v1.0.0

    Choose a part to bump version by:
    $ gto register nn --bump-minor
```

</details>

From the other hand, maybe we don't need to be smart and just move those examples to command-reference. And shorten UG pages, keeping links to commands instead of making small tutorials. WDYT?

Reply:
> I'm thinking if we need command-reference at all

At least a page listing all the commands and a description that goes a little beyond what `-h` output?

Not sure about the user guide. I'm trying to focus on the Get Started and cmd ref. TBH I think there's too much content for GTO docs.

Reply:
Re command reference: I've put together the first draft and polished the first three commands a bit (`annotate`, `assign`, `check-ref`). Could you please check them and give your feedback before I go to next ones?

Reply:
Accidentally I've added all other commands 😂 please check!

Reply:
@aguschin a full cmd ref review would be too much to manage here (again, I think the PR is too big). I only checked the index and first entry for now. Going back to the Get Started now.